metadata_tools/hosts/GO_0xxx/geometry_config.py

```
@@ -11,23 +11,108 @@
+                                                                      'METIS',
+                                                                      'ADRASTEA',
+                                                                      'AMALTHEA',
+                                                                      'THEBE'],
+                                                                                   ['SATURN'])]
+
+##########################################################################################
+# Exception functions
+##########################################################################################
+
+#=========================================================================================
+def except_test(observation):
```

Ruff (0.14.10)

Unused function argument: `observation`

(ARG001)